### ISSUE TITLE
Some globals aren't working

### ISSUE THREAD

**Issue by ajwood** [BIC-MNI/Register#10]
Some globals settings aren't working in the latest version of register. It looks like it might be everything that used to go in register.globals.

I get the impression that regsiter.globals isn't read at all; the variables that used to be defined there are still valid (i.e. they don't trigger an error message when they're in register_UI.globals), but setting them doesn't do anything.

For example:
`Slice_background_colour = BLACK;`

This doesn't produce an error, but also doesn't change the slice background colour.

**Comment by rdvincent** [comment]
The develop branch was changed to ignore register.globals a while ago. However, all of the variables should work. I'll investigate.

**Comment by ajwood** [comment]
I'm on the **release-1.0.08** branch of minc-toolkit, putting register at commit 317c9f091d12062ebb6bfd644cef510d6e137101

**Comment by rdvincent** [comment]
Yes, but I'm pretty sure that tag includes these changes. I'll have a fix in a few minutes.
